src/main/kotlin/model/DataPipelineTools.kt

```diff
@@ -0,0 +1,43 @@
+package uk.gov.justice.hmpps.architecture
+
+import com.structurizr.model.Model
+import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.ViewSet
+
+class DataPipelineTools private constructor() {
```

Comment:
Please see details in cec822d0926481f376cb728bec45fce7682a67d4

Reply:
We could use `DataMovingTools` as well, not sure which one sits better.

Reply:
This sock-drawer grouping of systems feels a little off. The three systems don't seem to interact, so this class isn't cohesive. Is there domain/author benefit to grouping them here (e.g. do we refer to them collectively as data pipeline tools already?). It _feels_ like the intention of `HMPPSSoftwareSystem` was to model a single system, so I'd be tempted to split these out into three classes.

My gut tells me we'll end up with quite a few systems which move or process data, so it _feels _ like this might get unwieldy fairly quickly. 

I'm not tightly coupled to this 🤪 so happy either way. We can easily refactor later on whichever route we choose now.

Reply:
Good points, thank you!

> Is there domain/author benefit to grouping them here (e.g. do we refer to them collectively as data pipeline tools already?)

No, you're right, they are not collectively referred to like this now.

I think the need to be able to identify similar components is in the air, though. But you're right, this kind of grouping is probably not the best. We could use tags, "problem areas", views 🤔 

I'll split this into separate systems.

Reply:
Done in various commits